stories/webapp/accountdashboard/WordPressIntegrationStory.js

```diff
@@ -0,0 +1,105 @@
+
+const ModalTrigger = withModalTrigger({
+  children: <Icon name="integration" />,
+  types: ['icon-only']
+});
+
+stories.add('WordPressIntegrationStory', () => {
+  const modalSubHeader = (
+    <Fragment>
+      <Icon name="integration" className="m-quarter mr-half" />
+      Wordpress v5.4
+      <span className="text-xs text-green-primary bg-green-90 px-quarter py-tenth inline-flex rounded-sm font-semi-bold mx-quarter">
+        CONNECTED
+      </span>
+      <span className="text-neutral-primary absolute right-0 mr-base">
+        Expires on 30/04/2020
+      </span>
+    </Fragment>
+  );
+
+  return (
+    <ModalTrigger>
+      <Modal.Container>
+        <Modal.Header text={modalSubHeader}>WordPress demo</Modal.Header>
+        <Modal.Body className="has-columns">
+          <div className="modal__column bg-neutral-98">
+            <div className="border border-solid border-neutral-90 rounded bg-white mb-half p-half flex items-center justify-between">
+              <label // eslint-disable-line jsx-a11y/label-has-associated-control
+                htmlFor="preview-url"
+                className="w-1/4 font-semi-bold ml-half"
+              >
+                Preview URL
+              </label>
+              <div className="flex-grow">
+                <InputWithButton
+                  inputId="preview-url"
+                  value="https://cms-wordpress.gatherpreview.com/acme"
+                  onClick={action('copy preview url')}
+                  // onClick={copyElementValueToClipboard}
+                  buttonText="Copy link"
+                  buttonTextAfterClick="Copied!"
+                />
+              </div>
+            </div>
+            <div className="border border-solid border-neutral-90 rounded bg-white mb-half p-half flex items-center justify-between">
+              <label // eslint-disable-line jsx-a11y/label-has-associated-control
+                htmlFor="preview-admin-url"
+                className="w-1/4 font-semi-bold ml-half"
+              >
+                Admin URL
+              </label>
+              <div className="flex-grow">
+                <InputWithButton
+                  inputId="preview-admin-url"
+                  value="https://cms-wordpress.gatherpreview.com/acme/wp-admin"
+                  onClick={action('copy admin url')}
+                  // onClick={copyElementValueToClipboard}
+                  buttonText="Copy link"
+                  buttonTextAfterClick="Copied!"
+                />
+              </div>
+            </div>
+            <div className="border border-solid border-neutral-90 rounded bg-white p-half flex items-center justify-between">
+              <label // eslint-disable-line jsx-a11y/label-has-associated-control
+                htmlFor="password"
+                className="w-1/4 font-semi-bold ml-half"
+              >
+                Password
+              </label>
+              <div className="flex-grow">
+                <InputWithButton
```

Comment:
🔴 If this is to be a password, then we should use a password type? Do we spread props over the input (🙏) and if so can we pass the password type in here?

Reply:
Yeah - we had this debate in 🐢. The point is it's not really a password - it's more of an access code, we're trying to make the site accessible, we want to make it easy to copy and paste/view this value, which is why we left it as visible text. The user is hopefully only going to copy the value and paste it in an email or chat message anyway